cg/services/analysis_starter/configurator/models/microsalt.py

```diff
@@ -0,0 +1,31 @@
+from pydantic import BaseModel
+
+from cg.constants import Workflow
+from cg.services.analysis_starter.configurator.abstract_model import CaseConfig
+
+
+class MicrosaltConfigContent(BaseModel):
+    CG_ID_project: str | None
+    Customer_ID_project: str | None
+    CG_ID_sample: str
+    Customer_ID_sample: str
```

Comment:
I don't remember why these attributes don't follow Pythonic name conventions. Is there a chance to change them?